bpf/bpf_host.c

```diff
@@ -1075,6 +1076,28 @@ static __always_inline int handle_l2_announcement(struct __ctx_buff *ctx)
 };
 #endif
 
+#if defined(ENABLE_IPSEC) && defined(ENABLE_IPSEC_RPS)
```

Comment:
Is there a benefit to omitting this in the preprocessor? Can the call site be guarded with `if CONFIG(enable_ipsec_rps) { ... }` instead? (see my other comments)